app/routes.py

```diff
@@ -1,2 +1,63 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, request, make_response
+
+planets_bp = Blueprint('planets', __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["POST", "GET"])
+def handle_all_planets():
+  if request.method == "POST":
+    request_body = request.get_json()
+    new_planet = Planet(name=request_body["name"], description = request_body["description"], circum = request_body["circum"])
```

Comment:
When there's a long-ish list of params, consider wrapping the lines to help keep things in a readable chunk.

Reply:
Be sure to include error handling to check whether the keys we are going to try to access are present. For instance, if we access the "name" key, and it's not in the request body, this will cause a `KeyError`, and crash the request. The default flask error handling returns the error messages as html, rather than json, so we should explicitly handle these situations ourselves, or look into how to customize the flask error handling.